### ISSUE TITLE
npm readmes should mention steps to take when `types` is set explicitly in the TS config

### ISSUE THREAD

**Issue by benediktwerner** [microsoft/TypeScript-DOM-lib-generator#1146]
When `types` is set explicitly in the TS config, the two simple installation steps described in the npm readmes (e.g. for [@types/web](https://github.com/microsoft/TypeScript-DOM-lib-generator/blob/main/deploy/readmes/web.md#installation)) aren't enough. You also need to add `"web"` to the `types` list. This should be mentioned in the READMEs.

**Comment by orta** [comment]
Yeah, makes sense - with TS 4.5 we'll have a much cleaner setup process too but adding that to the README can probably happen in a different PR.
